compose-builder/add-security.yml

```diff
@@ -54,8 +55,13 @@ services:
       - common-sec-stage-gate.env
     environment:
       SECRETSTORE_TOKENFILE: /tmp/edgex/secrets/edgex-security-bootstrap-redis/secrets-token.json
+      DATABASECONFIG_PATH: /redis/conf
+      DATABASECONFIG_NAME: redis.conf
+    tmpfs:
+      - /redis
```

Comment:
Suggest something more standards compliant like tmpfs: /run and put the redis conf there.

Reply:
good point, will change it to /run

Reply:
fixed